minetest/minetest
# Use performant and safe data structure for active objects

@Desour this was the idea I described

Note to bystanders: This won't make entities any faster than they were before (e.g. in 5.4.0), it will only walk back some recent performance regressions.

## To do

This PR is a Work in Progress.

- [x] make container work with `std::unique_ptr` (and resolve merge conflict)
- [x] find remaining bugs

## How to test

* Use entities and check that nothing blows up.
* try the test code from #13560
* ???
